Documentation/CrossPlatform/OculusQuestMRTK.md

```diff
@@ -0,0 +1,59 @@
+# How to configure Oculus Quest in MRTK using the XRSDK pipeline
+
+A [Oculus Quest](https://www.ultraleap.com/product/leap-motion-controller/) is required to use this data provider.
+
+The Oculus XRSDK Data Provider enables the use of the Oculus Quest with MRTK using [Unity's XR Pipeline](https://docs.unity3d.com/Manual/XR.html).
+This pipeline is the standard for developing XR applications in Unity 2019.3 and beyond.
+The Oculus Quest XR Plugin does not currently support hand tracking. In order to use hand tracking, consult the [MRTK-Quest](https://github.com/provencher/MRTK-Quest)
+project which uses the Legacy XR pipeline.
+
+## Deploying to Oculus Quest
+
+1. Follow [these steps](https://developer.oculus.com/documentation/unity/book-unity-gsg/) to ensure that your project is ready to deploy on Oculus Quest.
+
+1. Ensure that [developer mode] (https://developer.oculus.com/documentation/native/android/mobile-device-setup/) is enabled on your device. Installing the Oculus ADB Drivers is optional.
+
+1. Adding the Oculus XRSDK Data Provider
+    - Create a new Unity scene
```

Comment:
I dont think this step needs to be documented. Customers can use existing scenes.